pages/index.tsx

```diff
@@ -102,10 +110,6 @@ const Dashboard = () => {
         setLineOptions(lineOptions);
     };
 
-    useEffect(() => {
```

Comment:
I'm still new to NextJS, but I'm guessing this is the part where you are going to lock the tickets behind the authentication